packages/ui/scripts/generateCSS.ts

```diff
@@ -29,8 +30,11 @@ function writeCSS(props: {
     .process(result.css, { from: undefined })
     .then((result) => {
       result.warnings().forEach((warn) => {
+        // eslint-disable-next-line no-console
         console.warn(warn.toString());
       });
+      const dir = path.dirname(`dist/${outputPath}.css`);
+      fs.mkdirSync(dir, { recursive: true });
```

Comment:
Can we use `ensureDirSync` instead of `mkDirSync` here? The former is safer and handles edge cases more gracefully. 

```suggestion
      fs.ensureDirSync(dir);
```

Reply:
sure, done